### PR TITLE
[Rating] render non-integer/partial ratings

### PR DESCRIPTION
**Pull request by zombietfk**
## Description
Documentation updates for fomantic/Fomantic-UI#1074
 
## Screenshots
img

### CODE REVIEW

File: server/documents/modules/rating.html.eco [hunk 1]
```diff
@@ -83,6 +83,29 @@ themes      : ['Default']
       <br><br>
     </div>
 
+    <div class="example">
+      <h4 class="ui header">Partial Ratings </h4>
```
Comment (lubber-de) [inline server/documents/modules/rating.html.eco]: Please add `<div class="ui black label">New in 2.8.0</div>` before the `</h4>`  when your PR in the main repo gets merged

File: server/documents/modules/rating.html.eco [hunk 2]
```diff
@@ -83,6 +83,29 @@ themes      : ['Default']
...
+      <p>You can use decimal values for showing partially filled ratings</p>
+      <div class="ui ignored warning message">
+        <p>Partial ratings are only supported by browsers with support for <a href="https://caniuse.com/#feat=css-variables">CSS Variables</a> and <a href="https://caniuse.com/#feat=mdn-css_types_image_gradient_linear-gradient_interpolation_hints">Gradient Midpoints</a>. Unsupported browsers will use a rounded down value</p>
```
Comment (lubber-de) [inline server/documents/modules/rating.html.eco]: You should also add an info that jquery >3.2 is necessary as well (for people which use FUI from a CDN, they do not come across dependency warnings)

File: server/documents/modules/rating.html.eco (hunk 1 above)
Comment (hammy2899) [reply server/documents/modules/rating.html.eco]: @zombietfk bump

File: server/documents/modules/rating.html.eco (hunk 2 above)
Comment (hammy2899) [reply server/documents/modules/rating.html.eco]: @zombietfk bump

**Comment by hammy2899** [comment]
This has been merged

**Pull request closed without merging**